src/Maker/MakeRegistrationForm.php

```diff
@@ -138,9 +139,17 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
 
         $input->setArgument('will-verify-email', $willVerify);
 
+        // This must be preset to true to avoid code being generated if $willVerify === false
+        $input->setArgument('verify-email-with-auth', true);
+
         if ($willVerify) {
             $this->checkComponentsExist($io);
 
+            $input->setArgument(
+                'verify-email-with-auth',
+                $io->confirm('Do you want to require the user to be authenticated to verify their email?', true)
```

Comment:
Hmm, I'm trying to think about how a person would understand this... and the wording on it. Maybe we give some extra info before?

> To verify their email address, you can either require that users are authenticated when they click
> the verification link *or* allow them to be anonymous (e.g. maybe they click the link on a different
> device).

> Allow anonymous users to verify their email?

(and default to no)

Reply:
think we have this nailed down now